custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -38,6 +38,7 @@ def aggregation_query(self):
         ) (
           SELECT
             %(state_id)s AS state_id,
+            awcm.district_id,
```

Comment:
@dmydlo 
You mention you tested this locally but this this query looks wrong (you add a column to the `SELECT` but not the `INSERT` clause. Are you sure this query works the way that is intended?